packages/embed/README.md

```diff
@@ -159,6 +159,7 @@ Closing and opening a typeform in modal window will restart the progress from th
 | [onQuestionChanged](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/callbacks)                       | function                    | fires when user navigates between form questions                                                                                                                                                                                                                                                       | `undefined`                                                   |
 | [onHeightChanged](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/callbacks)                         | function                    | fires when form question height changes (eg. on navigation between questions or on error message)                                                                                                                                                                                                      | `undefined`                                                   |
 | [onEndingButtonClick](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/callbacks)                     | function                    | fires when button on ending screen is clicked, disables button redirect functionality                                                                                                                                                                                                                  | `undefined`                                                   |
+| onDuplicateDetected                                                                                                          | function                    | fires when the responder reaches the quota of submission defined in the duplicate prevention setting                                                                                                                                                                                                   | `undefined`                                                   |
```

Comment:
I would include a link to the help center page here: https://www.typeform.com/help/a/prevent-duplicate-responses-27917825492244/ and also slightly reword it ( _respondent_ is the correct term):

> fires when a respondent reaches the submission quota defined in [duplicate prevention setting](https://www.typeform.com/help/a/prevent-duplicate-responses-27917825492244/)

Reply:
Please update [docs in the repo](https://github.com/Typeform/embed/blob/main/docs/configuration.md#available-options) as well. It is deployed to https://developer.typeform.com/ automatically to update the docs in that place too.